HIP/hip-1249.md

```diff
@@ -189,8 +193,9 @@ Educate developers on key changes:
 
 1. **Simplified Billing**: No more 80% minimum charge—pay only for gas used. This reduces costs for overestimated limits
    and eases gas estimation.
-2. **Throttle Failures**: Transactions may still fail due to exhausted ops/sec (error: THROTTLED_AT_CONSENSUS), even if
-   valid. Advise retry logic and monitoring network load.
+2. **Throttle Failures**: A transaction may overfill the ops per second throttle up to a 15M gas per transaction limit.  
+   Subsequent transactions will fail (error: CONSENSUS_GAS_EXHAUSTED) until enough time passes to refill the throttle capacity, 
```

Comment:
Not until we refill all the throttle capacity, just until there is *some* throttle capacity.